skbio/alignment/_tabular_msa.py

```diff
@@ -2477,3 +2477,16 @@ def _is_sequence_axis(self, axis):
     @overrides(PositionalMetadataMixin)
     def _positional_metadata_axis_len_(self):
         return self.shape.position
+
+
+class TabularMSA2(TabularMSA):
```

Comment:
Can you clarify where this is coming from? Do we need this? Or can we just include this in `TabularMSA`?

Reply:
This will be included in `TabularMSA`, will address this in future commit.